lib/Models/GeoRssCatalogItem.js

```diff
@@ -0,0 +1,273 @@
+   * @type {Object}
+   */
+  updaters: {
+    get: function() {
+      return GeoRssCatalogItem.defaultUpdaters;
+    }
+  },
+
+  /**
+   * Gets the set of functions used to serialize individual properties in {@link CatalogMember#serializeToJson}.
+   * When a property name on the model matches the name of a property in the serializers object literal,
+   * the value will be called as a function and passed a reference to the model, a reference to the destination
+   * JSON object literal, and the name of the property.
+   * @memberOf GeoRssCatalogItem.prototype
+   * @type {Object}
+   */
+  serializers: {
+    get: function() {
+      return GeoRssCatalogItem.defaultSerializers;
+    }
+  },
+
+  /**
+   * Gets the data source associated with this catalog item.
+   * @memberOf GeoRssCatalogItem.prototype
+   * @type {DataSource}
+   */
+  dataSource: {
+    get: function() {
+      return defined(this._geoJsonItem)
+        ? this._geoJsonItem.dataSource
+        : undefined;
+    }
+  }
+});
+
+GeoRssCatalogItem.defaultUpdaters = clone(CatalogItem.defaultUpdaters);
+Object.freeze(GeoRssCatalogItem.defaultUpdaters);
+
+GeoRssCatalogItem.defaultSerializers = clone(CatalogItem.defaultSerializers);
+
+GeoRssCatalogItem.prototype._getValuesThatInfluenceLoad = function() {
+  return [this.url];
+};
+
+GeoRssCatalogItem.prototype._load = function() {
+  this._geoJsonItem = new GeoJsonCatalogItem(this.terria);
+  this._geoJsonItem.data = loadGeoRss(this);
+  this._geoJsonItem.name = this.name;
+  var that = this;
+
+  return this._geoJsonItem.load().then(function() {
+    if (!defined(that.rectangle)) {
+      that.rectangle = that._geoJsonItem.rectangle;
+    }
+  });
+};
+
+GeoRssCatalogItem.prototype._enable = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._enable();
+  }
+};
+
+GeoRssCatalogItem.prototype._disable = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._disable();
+  }
+};
+
+GeoRssCatalogItem.prototype._show = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._show();
+  }
+};
+
+GeoRssCatalogItem.prototype._hide = function() {
+  if (defined(this._geoJsonItem)) {
+    this._geoJsonItem._hide();
+  }
+};
+
+GeoRssCatalogItem.prototype.showOnSeparateMap = function(globeOrMap) {
+  if (defined(this._geoJsonItem)) {
+    return this._geoJsonItem.showOnSeparateMap(globeOrMap);
+  }
+};
+GeoRssCatalogItem.prototype.zoomTo = function() {
+  var that = this;
+  return when(this.load(), function() {
+    if (defined(that.nowViewingCatalogItem)) {
+      return that.nowViewingCatalogItem.zoomTo();
+    }
+
+    if (defined(that.rectangle)) {
+      return CatalogItem.prototype.zoomTo.call(that);
+    }
+
+    if (!defined(that.dataSource)) {
+      return;
+    }
+
+    return that.terria.currentViewer.zoomTo(that.dataSource);
+  });
+};
+
+function loadGeoRss(geoRssItem) {
+  return loadXML(proxyCatalogItemUrl(geoRssItem, geoRssItem.url)).then(xml => {
```

Comment:
In TerriaJS 7 and below we parse catalog item javascript files with an es5 parser to generate documentation for them.  Could you please replace this with the old anonymous function syntax `function (xml) {`?